### PR TITLE
now example works with 2.7 version of Symfony

### PR DESCRIPTION
**Pull request by ldath**

### CODE REVIEW

File: cache/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```
Comment (webmozart) [inline cache/.gitignore]: What's the purpose of this file?

**Comment by webmozart** [comment]
Thank you so much for this PR @ldath! :) This was long overdue.

File: cache/.gitignore (hunk 1 above)
Comment (ldath) [inline cache/.gitignore]: Sorry I forgot to delete. I normally use cache folder for twig templates, like this:

``` php
        // Set up Twig Environment with cache folder
        $environment = new Twig_Environment($loader, array(
            'cache' => $cache,
        ));
```

where $cache is path to Cache dir.

**Comment by ldath** [comment]
Old example is using deprecated method for CSRF. I'm using now this: 

``` php
use Symfony\Component\Security\Csrf\CsrfTokenManager;

...

// Prepare CSRF Token Manager for Forms
$csrf = new CsrfTokenManager();
$environment->addExtension(new FormExtension(new TwigRenderer($engine, $csrf)));

...
```

Those are examples from my separate project and differs from standalone-forms example.

I can prepare little more complicated Form example with `Callback` and `addEventListener` if there is a need.

**Comment by webmozart** [comment]
@ldath Yes, that would be much appreciated! But I'd do that in a separate PR. Can you remove the superfluous `.gitignore` file from this one? Then I will merge this.

**Comment by ldath** [comment]
Done

**Comment by webmozart** [comment]
Thanks! :)

**Pull request merged**